packages/flutter_test/lib/src/sheet_display.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+/// Place `cellNum` number of the target widget in a grid sheet.
+/// 
+/// The [SheetDisplay] tries to fill as much space as the parent allows, then
+/// display clones of the target widget in cells of size `cellSize`, placing
+/// cells from top left to bottom right, horizontal first. Cells are keyed from
+/// the tail to the front, meaning new clones are added to the front of the list.
+/// 
+/// See also:
+/// 
+///  * [WidgetTester.pumpFrames], which is often used together to display the
+///    frame-by-frame animation of a widget in a test.
+class SheetDisplay extends StatelessWidget {
+  /// Create an [SheetDisplay] using a fixed `target` widget.
+  /// 
+  /// The `cellSize` and `frameIndex` are required.
+  const SheetDisplay({
+    Key key,
+    this.target, 
+    @required this.cellSize,
+    @required this.cellNum,
+  }) : assert(cellSize != null),
+       assert(cellNum != null),
+       super(key: key);
+
+  /// The widget to show animation of.
+  final Widget target;
+
+  /// The size of a cell. Should contain a widget.
+  /// 
+  /// Should not change throughout the test.
```

Comment:
Why? The widget would just re-layout with the changed size, no?

Reply:
I've removed this requirement.